tests/runtimes/test_application.py

```diff
@@ -190,6 +190,43 @@ def test_application_api_gateway(rundb_mock):
     assert function_name in api_gateway.spec.functions[0]
 
 
+def test_application_runtime_resources(rundb_mock):
+    mlrun.mlconf.igz_version = "3.6.0"
```

Comment:
why is it needed? also, will it not corrupt consecutive tests?

Reply:
It is due to this: https://github.com/mlrun/mlrun/blob/5d8c6d707d4711c02ae52ee4a714cf2ec0102a6e/mlrun/runtimes/nuclio/api_gateway.py#L493-L494 

I will add a setup/teardown with resetting the version to the original

Reply:
@liranbg added fixture to avoid corruption